cmd/cerbosctl/get/get.go

```diff
@@ -4,50 +4,150 @@
 package get
 
 import (
-	"context"
 	"fmt"
-	"io"
 
 	"github.com/spf13/cobra"
 
-	policy "github.com/cerbos/cerbos/api/genpb/cerbos/policy/v1"
 	"github.com/cerbos/cerbos/client"
 	"github.com/cerbos/cerbos/cmd/cerbosctl/internal"
 )
 
-var getPolicyFlags = internal.NewGetPolicyFilterDef()
+const (
+	getExample = `# List derived_roles
+cerbosctl get derived_roles
+cerbosctl get derived_role
+cerbosctl get dr
+
+# List and filter derived_roles
+cerbosctl get derived_roles --name my_derived_roles
+
+# List principal policies
+cerbosctl get principal_policy
+cerbosctl get principal_policies
+cerbosctl get pp
+
+# List and filter principal policies
+cerbosctl get principal_policies --name donald_duck --version default
+
+# List resource policies
+cerbosctl get resource_policy
+cerbosctl get resource_policies
+cerbosctl get rp
+
+# List and filter resource policies
+cerbosctl get resource_policies --name leave_request --version default
+
+# List all schemas
+cerbosctl get schema
+cerbosctl get schemas
+cerbosctl get s
+
+# Get policy
+cerbosctl get policy derived_roles.my_derived_roles -oyaml
+
+# Get policy (disk, git, blob stores)
+cerbosctl get policy blog_derived_roles.yaml principal.yaml
+
+# Get policy (mutable stores)
+cerbosctl get policy derived_roles.my_derived_roles`
+)
+
+type Arguments struct {
+	Output    string
+	Name      []string
+	Version   []string
+	NoHeaders bool
+}
+
+var getArgs = &Arguments{}
 
 func NewGetCmd(fn internal.WithClient) *cobra.Command {
 	cmd := &cobra.Command{
-		Use:   "get",
-		Short: "Get policy",
-		RunE:  fn(runListCmdF),
+		Use:     "get",
+		Short:   "Get",
+		Example: getExample,
+		RunE:    fn(runGetCmd),
 	}
 
-	cmd.Flags().AddFlagSet(getPolicyFlags.FlagSet())
+	cmd.Flags().BoolVar(&getArgs.NoHeaders, "no-headers", false, "Print no headers")
+	cmd.Flags().StringVarP(&getArgs.Output, "output", "o", "json", "Output format for the policies; json, yaml, prettyjson formats are supported")
```

Comment:
Let's make the default YAML because that's what users usually deal with
```suggestion
	cmd.Flags().StringVarP(&getArgs.Output, "output", "o", "yaml", "Output format for the policies; json, yaml, prettyjson formats are supported")
```